README.md

```diff
@@ -1,20 +1,453 @@
+
+## Creating Objects
+
+So far we've seen one way to create an Object but there are a few more.
+
+
+#### Using Object literals
+In our Pokemon example, we created an Object using curly braces. This is called the Object literal syntax:
+
+```js
+var obj = {};
+```
+
+Objects created using the Object literal syntax can have key-value pairs passed in between the braces as seen in the pikachu example above or have values added to them after creation as we'll see later on in the [Adding Properties To Objects](#adding-properties-to-objects) section.
+
+#### Using the Object constructor
+
+The keyword `Object` is a function that when invoked, returns an instance to a new Object.
+
+```js
+var obj = new Object();
+```
+
+#### Using Object.create method
+
+`Object.create` is a method for creating Objects with a specific prototype. Passing `null` will create an Object that has no prototype. This can be useful when we don't care about inherting properties from the prototype chain.
+
+```js
+var obj = Object.create(null);
+```
+
+## Adding properties to Objects
+
+By default, Objects are mutable. This means that after an Object is created, we can add additional properties to it at any time. There are a few ways to add a property to an Object:
+
+#### Object literals
+
+Object can be defined with a set of keys and values using the Object literal syntax
+
+```js
+var pikachu = {
+	"name": "Pikachu"
+};
+```
+
+When using this syntax, keys don't have to be wrapped in quotes. The following is also valid:
+
+```js
+var pikachu = {
+	name: "Pikachu" // Notice the name key isn't wrapped in quotes
+};
+```
+
+#### Key Indexes
+
+With key indexes, we use the Object's variable name, followed by the key string we want to use wrapped in square brackets. This is very similar to accesing an element in an Array via its index number.
+
+```js
+var pikachu = {};
+pikachu["name"] = "Pikachu";
+```
+
+We could also use a string variable as the key:
+
+```js
+var prop = "name";
+
+pikachu[prop] = "Pikachu"; // same as pikachu["name"] = "Pikachu"
+```
+
+Since we left out the quotes in [prop], we used the value of the prop variable which is "name".
+
+#### Dot notation
+
+With dot notation, we use the Object's variable name with a dot at the end, followed by the name of the key we want to use:
+
+```js
+var pikachu = {};
+pikachu.name = "Pikachu";
+```
+
+Not all key names can be used with dot notation. For example, the following is invalid JavaScript syntax:
+
+```js
+pikachu.max-level = 100; // this is invalid
+```
+
+This is invalid because variable names can't have hyphens. If we wanted to use a key that doesn't follow JavaScript's naming constraints for variables, we'd have to use the key index since strings can hold any character:
+
+```js
+pikachu["max-level"] = 100; // this is valid
+```
+
+#### Object.defineProperty
```

Comment:
This section down to the section on `Object.freeze` could, I think, be broken out into a separate lesson (or separate lessons).